packages/react-codemirror-playground/src/App.tsx

```diff
@@ -112,10 +116,20 @@ export function App() {
               theme={darkMode ? 'dark' : 'light'}
               history={Object.values(demos)}
               schema={schema}
-              featureFlags={{ signatureInfoOnAutoCompletions: true }}
+              featureFlags={{
+                signatureInfoOnAutoCompletions: true,
+                format: formatFlag,
```

Comment:
Do you feel we need a feature flag at the library level @ncordon? To me it'd be fine enough with only a feature flags at the VSCode / Query level. In query the keyboard command would still work, but I don't think people would accidentally run into that and if they did, i don't think there's any hard done. It'd be more about not advertising

Reply:
This is fine to me, the library is still at experimental versions